Services/Maps/CarsToPinsConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Models.Cars;
+using Services.WebAPI.Cars;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms.Maps;
+
+namespace Services.Maps
+{
+    public class CarsToPinsConverter
```

Comment:
One method one responsibility - do await service.GetAll()  outside of CarsToPinsConverter class (in MapModel), then convert list to List< Pin >.